dicomImport/matRad_importDicomSteeringParticles.m

```diff
@@ -137,25 +137,61 @@
         c1_help = currContrSeq.ScanSpotPositionMap(1:2:(2 * numOfScanSpots));
         c2_help = currContrSeq.ScanSpotPositionMap(2:2:(2 * numOfScanSpots));
         weight_help = currContrSeq.ScanSpotMetersetWeights;
-        StfTmp = [StfTmp; c1_help c2_help (currEnergy * ones(numOfScanSpots,1)) weight_help (currFocus * ones(numOfScanSpots,1))];
+        if isfield(currContrSeq, 'RangeShifterSettingsSequence')
+            % rangeshifter identification
+            rashiID = currContrSeq.RangeShifterSettingsSequence.Item_1.ReferencedRangeShifterNumber;
```

Comment:
Hm, technically _we_ don't need / process it. I thought we might keep it like it is in the DICOM world.